core/templates/domain/collection/collection-node.model.spec.ts

```diff
@@ -45,30 +49,20 @@ describe('Collection node model', () => {
       category: 'Algebra',
       title: 'exp title'
     };
-    var collectionNodeBackendObject = {
+
+    var collectionNode = CollectionNode.create({
       exploration_id: 'exp_id0',
       exploration_summary: explorationSummaryBackendObject
-    };
-
-    var collectionNode = CollectionNode.create(
-      collectionNodeBackendObject);
+    });
     expect(collectionNode.getExplorationId()).toEqual('exp_id0');
     expect(collectionNode.getExplorationTitle()).toEqual('exp title');
 
-    var summaryObject = collectionNode.getExplorationSummaryObject();
+    var summaryObject: SummaryObject =
```

Comment:
Why `SummaryObject`, why not `LearnerExplorationSummaryBackendDict` which is returned by the `getExplorationSummaryObject`?

Reply:
I'll update this to use `LearnerExplorationSummaryBackendDict` instead. The category field in `LearnerExplorationSummaryBackendDict` will still need to be marked optional, and the category field type will be string | undefined in LearnerExplorationSummary.

Reply:
Hmmm, actually the solution with custom type seems better, could we just name the type in a way so that it is understandable that it belongs only into a spec file?